tests/environment/.env

```diff
@@ -1,4 +1,4 @@
 TRANSFORMERS_TAG=2.1.1
-SOLR6_TAG=1.4.0
+SOLR6_TAG=1.4.1
```

Comment:
This was automatically brought into the PR when creating. This change was added to master in #510

Reply:
This PR should not bring additional and unintentional changes

Reply:
Removed unintentional changes